### ISSUE TITLE
Error while building theme slick-theme.css

### ISSUE THREAD

**Issue by KrzysztofMadejski** [bezbeli/faktograf#1]
During `gulp` I'm getting `Unrecognised input in file /home/kmadejski/Projects/faktograf/wp-content/themes/faktograf/bower_components/slick-carousel/slick/slick-theme.css line no. 33`

The effect is: I'm not getting main.css in the dist folder.

```
$ bower list
(node:10177) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
bower check-new     Checking for new versions of the project dependencies...
sage /home/kmadejski/Projects/faktograf/wp-content/themes/faktograf
├── blueimp-gallery#2.16.0 (latest is 2.21.3)
├─┬ bootstrap-sass#3.3.5 (latest is 3.3.7)
│ └── jquery#3.1.1
├── font-awesome#4.4.0 (latest is 4.6.3)
├─┬ imagesloaded#3.1.8 (latest is 4.1.1)
│ ├── eventEmitter#4.3.0 (latest is 5.1.0)
│ └── eventie#1.0.6
├─┬ jquery-ias#2.2.2
│ └── jquery#3.1.1
├─┬ slick-carousel#1.5.9 (latest is 1.6.0)
│ └── jquery#3.1.1
└─┬ velocity#1.2.3 (latest is 1.3.1)
  └── jquery#3.1.1
```

**Comment by KrzysztofMadejski** [comment]
I've set slick fixed  to 1.5.8 and got new error

```
'layouts/_tinymce.less' wasn't found. Tried - /home/kmadejski/Projects/faktograf/wp-content/themes/faktograf/assets/styles/layouts/_tinymce.less,layouts/_tinymce.less in file /home/kmadejski/Projects/faktograf/wp-content/themes/faktograf/assets/styles/main.scss line no. 24
```

https://github.com/roots/sage/issues/1648 ?

**Comment by KrzysztofMadejski** [comment]
Ok, bumping version of gulp-if and downgrading slick helped.
